config/services.yaml

```diff
@@ -49,6 +49,3 @@ services:
     Centreon\Application\Controller\Router:
         decorates: 'router'
         arguments: ['@Centreon\Application\Controller\Router.inner']
-
-imports:
-    resource: './Modules/*.yaml'
```

Comment:
not needed anymore ?

Reply:
Exactly, because Symfony will automatically load the configuration files depending on the environment.
https://symfony.com/doc/current/configuration.html#configuration-environments